### PR TITLE
Redux set up

### PR DESCRIPTION
**Pull request by J-C-S-V**
Hello dear reviewer 👋, I hope you're having a nice day today 😊

This is what I did on this branch:

- Add Redux Toolkit
- Create a directory that will contain all your Redux logic
- Configure a Redux store
- Define a slice of state for books
- Define a slice of state for categories

Best regards! ✨

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+  },
+});
```
Comment (rloterh) [inline src/redux/books/booksSlice.js]: - [x] Let's be reminded that in _bookSlice_, we are required to create two reducers, namely  `addBook` and `removeBook`. Let's, therefore, ensure to add the `removeBook` reducer.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Juank, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!
🏁 🚦 ⛪ 🚜 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Good job so far adding reducers and actions!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly include appropriate instructions in your ReadMe on how to _setup_ and _run_ your app, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (J-C-S-V) [reply src/redux/books/booksSlice.js]: Hello @rloterh,

You're right, thanks for the reminder, I'm going to add that reducer right away 😊

Regards!

File: src/components/BookCard.jsx [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+    { title: 'The Hunger Games', author: 'Suzanne Collins' },
+    { title: 'To Kill a Mockingbird', author: 'Harper Lee' },
+    { title: 'The Catcher in the Rye', author: 'J.D. Salinger' },
+  ];
+
+  return (
+    <div className="book-list">
+      {/* {booksDescription.map((book) => (
+        <Book key={book.title} title={book.title} author={book.author} />
+      ))} */}
```
Comment (BregornOriginal) [inline src/components/BookCard.jsx:15]: [SUGGESTION and OPTIONAL for now] Your project now is approved, but kindly remove the commented code and remember to do this in your next ones, so you can maintain a cleaner project 💯

**Review by BregornOriginal — APPROVED** [review]
Hi @J-C-S-V 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy (2)

## Highlights

✔️ Linters are passing
✔️ Good commit history
✔️ Good work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Comment by J-C-S-V** [comment]
Thank you!!!!! @BregornOriginal ✨

**Pull request merged**